### DISCUSSION TITLE
Do anyone need to generate LRC or KAR files?

### DISCUSSION THREAD

**Discussion by rakuri255** [rakuri255/UltraSinger#49] [Q&A]
UltraSinger could generate LRC and KAR files.
But do anyone need these?

**Comment by rogerGunis** [comment]
Ultrastar can read these files. Thus it is an alternative as I understand. Txt files are the most used type on internet and editors. Ultrastar cannot generate kar/lrc files itself.

https://www.mymusictools.com/de/karaoke_software_19/karafun_karaoke_player_20220.htm
Karafun is using these types. Thus if you want to extend your fan base ... :-)

**Comment by eadmaster** [comment]
ASS files can be converted with this:

https://github.com/SamLangTen/ASS2LRC

Edit: even SubtitleEdit can do the conversion  https://github.com/SubtitleEdit/subtitleedit

**Comment by rakuri255** [comment]
What is the differance with .kar + midi an just an midi that has lyrics?
